extensions_admin/test/unit/extensions/admin/upload/test_comps.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import mock
+
+from pulp.client.commands.repo.upload import UploadCommand
+
+from pulp_rpm.extensions.admin.upload import comps
+from pulp_rpm.common.ids import TYPE_ID_PKG_GROUP
+from pulp_rpm.devel.client_base import PulpClientTests
+
+
+DATA_DIR = os.path.abspath(os.path.dirname(__file__)) + '/../../../../../../plugins/test'
```

Comment:
Austin wrote:
> I've seen this elsewhere, but it doesn't seem ideal. @rbarlow
> <https://github.com/rbarlow>, do you know of a better way?

Eh, for tests I don't think it's so bad. But one option could be to put
the DATA_DIR variable inside pulp_rpm.devel.somewhere and then you could
import that instead.